editor/editor_settings.cpp

```diff
@@ -562,6 +568,10 @@ void EditorSettings::_load_defaults(Ref<ConfigFile> p_extra_config) {
 	_initial_set("filesystem/on_save/compress_binary_resources", true);
 	_initial_set("filesystem/on_save/safe_save_on_backup_then_rename", true);
 
+	// File server
+	_initial_set("filesystem/file_server/port", 6010);
```

Comment:
Should this have a range?

Reply:
Yes but with `no_slider`.

Reply:
Yeah there should probably be a range if there's a min and max for valid ports. I looked through the code a bit and we don't seem to do any validation for the port range currently in EditorFileServer/TCPServer/NetSocketPosix before passing it to the OS APIs.

CC @Faless @mhilbrunner